packages/headless/src/features/commerce/facets/facet-search-set/regular/commerce-regular-facet-search-request-builder.ts

```diff
@@ -42,3 +42,9 @@ export const buildFacetSearchRequest = (
     ...(!isFieldSuggestionsRequest && {...restOfCommerceAPIRequest, query: ''}),
   };
 };
+
+export function removeCommerceFieldSuggestionNamespace(facetId: string) {
```

Comment:
I'd rename this to `getFacetIdWithoutFieldSuggestionNamespace` (because IMO, the verb "remove" suggests that the function will mutate the argument, which it doesn't)

Reply:
Also, could we move this function to `commerce-facet-search-actions.ts`? I think we could modify the original facetId directly in `getExecuteFacetSearchThunkPayloadCreator` instead of doing it inside the request builder functions themselves.

Reply:
We do it in the request builder since the `facetId` can be namespaced (or not) here: https://github.com/coveo/ui-kit/blob/162299b9084351e064ccb7da63135f6457773d72/packages/headless/src/features/commerce/facets/facet-search-set/regular/commerce-regular-facet-search-request-builder.ts#L12

Doing it in `getExecuteFacetSearchThunkPayloadCreator` would require conditionally namespacing the `facetId.